build/devbase/godeps.sh

```diff
@@ -32,6 +32,7 @@ ${GOPATH}/bin/glock sync github.com/cockroachdb/cockroach
 pkgs="
 github.com/biogo/store/interval
 github.com/biogo/store/llrb
+github.com/cenkalti/backoff
```

Comment:
The docker client is used by our acceptance tests which we don't run in CI.
I think we just neglected to update this file.
On Jun 15, 2015 11:23, "Ben Darnell" <notifications@github.com> wrote:

> In build/devbase/godeps.sh
> <https://github.com/cockroachdb/cockroach/pull/1364#discussion_r32431543>:
>
> > @@ -32,6 +32,7 @@ ${GOPATH}/bin/glock sync github.com/cockroachdb/cockroach
> >  pkgs="
> >  github.com/biogo/store/interval
> >  github.com/biogo/store/llrb
> > +github.com/cenkalti/backoff
>
> This seems like a very heavyweight dependency for a small bit of
> functionality if it pulls in three other libraries (including a docker
> client?). I think we'd need a very compelling reason to add a dependency
> for this instead of implementing a simple exponential backoff ourselves.
> Remember that the retry system is used in the client so it will be linked
> into client apps in go and we will need to replicate its functionality in
> all other languages.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/1364/files#r32431543>.
>